server/tests-py/queries/v1/update/basic/person_error_no_where_clause.yaml

```diff
@@ -3,7 +3,7 @@ url: /v1/query
 status: 400
 response:
   path: $
-  error: key "where" not present
+  error: key "where" not found
```

Comment:
Out of interest: where do these changed error messages come from? Are they caused by the fact that you now raise these errors through `throw400` via `Monadfail`, rather than constructing the errors explicitly? In any case, to me the new messages look more informative.

Reply:
These changes just come from a newer aeson I believe, glad you think they are an improvement